ai-service/README.md

```diff
@@ -0,0 +1,33 @@
+# Liberty-Bikes External AI
+
+
+## Join a game:
+
+```
+http://localhost:8083/open/ws/{partyID}
+```
+
+## Local development
```

Comment:
this needs to be updated for running the AI service. Could just delete this README for now and then we can add the relevant doc to the primary README later